src/dom/rect.js

```diff
@@ -234,21 +247,63 @@ export default class Rect {
 	}
 
 	/**
+	 * Checks if all properties ({@link #top}, {@link #left}, {@link #right},
+	 * {@link #bottom}, {@link #width} and {@link #height}) are the same as in the other `Rect`.
+	 *
+	 * @param {module:utils/dom/rect~Rect} rect A `Rect` instance to compare with.
+	 * @returns {Boolean} `true` when Rects are equal. `false` otherwise.
+	 */
+	isEqual( anotherRect ) {
+		for ( const prop of rectProperties ) {
+			if ( this[ prop ] !== anotherRect[ prop ] ) {
+				return false;
+			}
+		}
+
+		return true;
+	}
+
+	/**
+	 * Checks whether a `Rect` fully contains another `Rect` instance.
```

Comment:
The same issue as everywhere else – I don't think that it makes sense to abuse preformatted text and type names. IMO, this makes reading the descriptions harder because the text is less heterogeneous. Plus, it's semantically incorrect as well because you're not checking `Rect` (type) here but a rect instance.

For me, it should be: "Checks whether this rect fully contains another rect instance."

The same applies to many other places in these docs.